pkg/config/config.go

```diff
@@ -57,9 +57,12 @@ var (
 	modelServerService = fmt.Sprintf("kepler-model-server.%s.cluster.local", KeplerNamespace)
 
 	KeplerNamespace              = getConfig("KELPER_NAMESPACE", defaultNamespace)
-	EnabledEBPFCgroupID          = false
-	ExposeHardwareCounterMetrics = true
-	EnabledGPU                   = false
+	EnabledEBPFCgroupID          = getBoolConfig("ENABLE_EBPF_CGROUPID", true)
```

Comment:
if we use getConfig and getBoolConfig I suppose we'd better moving them into an init function.
to avoid invoke functions too many times?
with an init function I suppose we have chance to init all vars in once time?

Reply:
Sound good to me. I will move it.

Reply:
getConfig/getBoolConfig will be called just once for each variable at initial point. If the cmd/exporter.go has an conflict setting it will just applied the value already initialized.